R/calculate_silhouette_width.R

```diff
@@ -0,0 +1,76 @@
+#' Calculate silhouette width scores from an integrated SCE object
```

Comment:
We should probably expand this description a bit to say that this is performing replicated calculations on downsampled datat